src/steps/iPressKey.js

```diff
@@ -0,0 +1,50 @@
+module.exports = {
+	docs: {
+		description: "Fire a key press",
+		synopsis: "I press <key> [+<modifier>] at <control> [in <viewName> view]",
```

Comment:
better and more consistent: I press KEY [+ MODIFIER] on CONTROL_ID [in VIEW_NAME view]

- also note that usually modifiers read better if placed before the key. So better would be MODIFIER + KEY. E.g., ALT+X not X+ALT.